suite-common/wallet-core/src/device/deviceReducer.ts

```diff
@@ -514,9 +516,28 @@ export const selectDeviceRequestedPin = (state: DeviceRootState) => {
     return buttonRequestsCodes?.some(code => code?.startsWith('PinMatrixRequestType')) ?? false;
 };
 
+export const selectIsUnacquiredDevice = (state: DeviceRootState) => {
+    const deviceType = selectDeviceType(state);
+    return deviceType === 'unacquired';
+};
+
 export const selectIsDeviceInitialized = (state: DeviceRootState) => {
     const features = selectDeviceFeatures(state);
-    return !!features?.initialized;
+    const mode = selectDeviceMode(state);
+
+    if (!features?.initialized) return false;
+
+    if (mode === 'initialize' || mode === 'seedless') return false;
+
+    return true;
+};
+
+export const selectIsDeviceConnectedButNotInitialized = (state: DeviceRootState) => {
+    const device = selectDevice(state);
+    const isUnacquiredDevice = selectIsUnacquiredDevice(state);
+    const isDeviceInitialized = selectIsDeviceInitialized(state);
+
+    return device && !isUnacquiredDevice && !isDeviceInitialized;
```

Comment:
```suggestion
    return  !isUnacquiredDevice && !isDeviceInitialized;
```

check for `device` is redundant. If there is no device, then both the `isUnacquiredDevice` and `isDeviceInitialized ` will be `false`.

Reply:
It is needed there but I simplified it a little bit.
https://github.com/trezor/trezor-suite/pull/9625/commits/ab744c7ac919ec4bf4231ea97b52ee8c10af56e1

Reply:
+ rename [26ad63a](https://github.com/trezor/trezor-suite/pull/9625/commits/26ad63a185be5825ba7cddd82b637b7c0514bf3c)